wsgiservice_restplus/namespace.py

```diff
@@ -285,11 +339,14 @@ def wrapper(documented):
                 documented._validations = {}
             documented._validations[name] = {'re': re, 'convert': convert, 'doc': doc, 'mandatory': mandatory}
```

Comment:
Interesting that here we have this solution and above we use `field.__apidoc__ = merge(getattr(field, '__apidoc__', {}), {'as_list': True})` rather.... I don't know which is better.... just interesting.